packages/contracts/test/contracts/L1/messaging/L1StandardBridge.spec.ts

```diff
@@ -1,57 +1,31 @@
-/* External Imports */
 import { ethers } from 'hardhat'
-import { Signer, ContractFactory, Contract, constants } from 'ethers'
-import { Interface } from 'ethers/lib/utils'
-import {
-  smock,
-  MockContractFactory,
-  FakeContract,
-  MockContract,
-} from '@defi-wonderland/smock'
-
-/* Internal Imports */
```

Comment:
Not a big deal, but I liked these 'layout annotation' comments.

Reply:
We recently added a linting change that automatically sorts/separates imports. Kinda makes the comments unnecessary now. The comments can also cause issues where the linter isn't able to automatically organize the imports. I think it's fine now that it's enforced automatically.